tests/test_loaders.py

```diff
@@ -46,9 +46,7 @@
 
 
 def create_remote_index(httpserver, dataset_name):
-    httpserver.serve_content(
-        open(REMOTE_DATASETS[dataset_name]["local_index"], "rb").read()
-    )
+    httpserver.serve_content(open(REMOTE_DATASETS[dataset_name]["local_index"], "rb").read())
```

Comment:
just to be sure - did you change any code in this file? it looks like this is all just reformatting

Reply:
This is all just reformatting.